static/js/brand-store/SectionNav/SectionNav.test.tsx

```diff
@@ -5,7 +5,7 @@ import "@testing-library/jest-dom";
 import SectionNav from "./SectionNav";
 
 jest.mock("react-router-dom", () => ({
-  ...jest.requireActual("react-router-dom"),
+  ...jest.requireActual<any>("react-router-dom"),
```

Comment:
Could you give more details on the TS error message here please?

If you don't want to create a proper type/interface now, might be worth creating a type to be updated in the future as more files get moved over.

As an example we use the following in maas-ui:
`export type TSFixMe = any;`

Reply:
The error was `Spread types may only be created from object types`. I've removed the type now and there is no error so I think my editor might have not updated.

Reply:
Awesome 👍